README.md

```diff
@@ -485,14 +488,14 @@ You may also use the `{repeater}` tag for repeating content:
         </div>
     {/repeater}
 
-For more details on syntax fields, see the [Parser section](http://octobercms.com/docs/services/parser#dynamic-syntax-parser) of the October documentation.
+For more details on syntax fields, see the [Parser section](https://octobercms.com/docs/services/parser#dynamic-syntax-parser) of the October documentation.
 
 ##### Custom menu item form fields
 
 Just like CMS objects have the view bag component to store arbitrary values, you may use the `viewBag` property of the `MenuItem` class to store custom data values and add corresponding form fields.
 
-    Event::listen('backend.form.extendFields', function ($widget) {
-
+    Event::listen('backend.form.extendFields', function ($widget)
+    {
```

Comment:
@bennothommo please don't accept coding style changes for no reason. Also, I would prefer that closures and other inline functions have their opening brace on the same line as their definition while methods have their opening brace on a newline.

Reply:
@LukeTowers My apologies. I had reviewed it and it all looked good, but missed that one. I've fixed it in afdbb1d4b97fc0914d8fe16ad548a19c7fbc0319.

Reply:
No problem, you're doing a stellar job!